components/clarity-vscode/client/src/clientBrowser.ts

```diff
@@ -19,7 +19,7 @@ export async function activate(context: ExtensionContext) {
     serverWorkerReady = resolve;
     workerTimeout = setTimeout(() => {
       reject(new Error("worker timeout"));
-    }, 5000);
+    }, 10000);
```

Comment:
On (very) slow networks, the server can take a few seconds to load, to be safe let's increase the delay before throwing an error